TESTS/mbed_platform/minimal-printf/compliance/main.cpp

```diff
@@ -51,6 +51,8 @@ static control_t test_printf_d(const size_t call_count)
 
     /*************************************************************************/
     /*************************************************************************/
+#if !defined(__NEWLIB_NANO)
+    //Printf with %hhd is not supported by Newlib-Nano
```

Comment:
```suggestion
    // The format specifier %hhd is not supported by Newlib-Nano
```
The same comment everywhere else.